src/app/products.php

```diff
@@ -1 +1,138 @@
-<?php
+<?php require_once '../../vendor/autoload.php'; ?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Totally Pawesome</title>
+    <link rel="stylesheet" type="text/css" href="../assets/css/vendor/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="../assets/css/styles.css">
+</head>
+<body>
+<section class="row banner">
+    <div class="home col-xs-2">
+        <a href="#">
+            <h2>Home</h2>
+        </a>
+    </div>
+    <div class="shop-name col-xs-10">
+        <h2>Totally Pawesome</h2>
+    </div>
+</section>
+<section class="row main">
+    <div class="sidebar col-xs-2">
+        <a href="#">
+            <h4>Category1</h4>
+        </a>
+        <a href="#">
+            <h4>Category2</h4>
+        </a>
+        <a href="#">
+            <h4>Category3</h4>
+        </a>
+        <a href="#">
+            <h4>Category4</h4>
+        </a>
+        <a href="#">
+            <h4>Category5</h4>
+        </a>
+        <a href="#">
+            <h4>Category6</h4>
+        </a>
+    </div>
+    <div class="main-content col-xs-10">
+        <div class="row">
+            <h2>Product Category</h2>
+        </div>
+        <div class="row">
+            <div class="product col-xs-8 col-sm-6 col-md-4 col-lg-3">
```

Comment:
These are showing 2 blocks across, the wireframe has 4 blocks across.

Reply:
May be a local issue, mine is showing 4